kv/kvnemesis: TestKVNemesisSingleNode failed

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://tanzanite.cluster.engflow.com/invocations/default/c3af9552-3cdb-4ea6-b57c-71957ce8d83e?testReportRun=3&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZuZW1lc2lzOmt2bmVtZXNpc190ZXN0) on master @ [3c41c509a87cba7a1fd3f5cfdb0f6badb78e3704](https://github.com/cockroachdb/cockroach/commits/3c41c509a87cba7a1fd3f5cfdb0f6badb78e3704):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestKVNemesisSingleNode (14m55s)

```
Stack: 

```
goroutine 157188 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2259 +0x3b9
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestKVNemesisSingleNode3915654839
    test_log_scope.go:81: use -show-logs to present logs inline
    kvnemesis_test.go:328: seed: 2655567903333817945
    kvnemesis_test.go:235: kvnemesis logging to /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/testlogs/pkg/kv/kvnemesis/kvnemesis_test/run_3_of_30/test.outputs/kvnemesis3391632818
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #118732 kv/kvnemesis: TestKVNemesisSingleNode failed [C-test-failure O-robot T-kv branch-release-22.2 release-blocker]
</p>
</details>
/cc @cockroachdb/kv
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/kv/kvnemesis/BUILD.bazel
+++ b/pkg/kv/kvnemesis/BUILD.bazel
@@ -60,7 +60,7 @@ go_library(
 
 go_test(
     name = "kvnemesis_test",
-    size = "large",
+    size = "medium",
     srcs = [
         "applier_test.go",
         "engine_test.go",
@@ -74,7 +74,7 @@ go_test(
     embed = [":kvnemesis"],
     exec_properties = select({
         "//build/toolchains:is_heavy": {"test.Pool": "heavy"},
-        "//conditions:default": {"test.Pool": "default"},
+        "//conditions:default": {"test.Pool": "large"},
     }),
     deps = [
         "//pkg/base",

